aqw-opcua-server/Settings.h

```diff
@@ -1,24 +1,24 @@
 #pragma once
-#include "cpprest/http_client.h"
+
 #include <fstream>
 #include <iostream>
 
-namespace weathersvr {
+#include <cpprest/http_client.h>
+
+namespace weatherserver {
 
     class Settings {
 
         public:
 
             Settings();
-            /*
-            Open the file passed to the formal parameter fileName and set the settings to variables of this class.
-            @param char* fileName - the path containing the settings file name and extension that is passed to command line arguments.
-            */
-            bool setup(char* fileName);
+
+            //Open the SETTINGS_FILE_NAME file and assign values to variables.
+            bool setup();
```

Comment:
We can read settings from file in constructor and set flag so that client code can use it like this:
```c++
// extract path to executable
std::string pathToExeDir=...;
// initialize settings
Settings settings(pathToExeDir);
if (!settings.isValid())
    return EXIT_FAILURE;
```